### PR TITLE
Nlopt.0.6.1

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
Cc @mkur.

**Comment by UnixJunkie** [comment]
The previous package version was wrong (extra leading 0.).
The tarball of the package is now hosted by the package creator since a PR was merged (we can now opam pin the official repository).
Sorry for any misunderstanding.
And indeed, there was some package renaming (not due to me): previously this package was nlopt-ocaml, but it became nlopt recently.

**Comment by UnixJunkie** [comment]
related to https://github.com/ocaml/opam-repository/pull/21744

**Comment by kit-ty-kate** [comment]
Looks good. Could you add a `nlopt-ocaml.transition` package (similar to `jbuilder.transition`) so that the move from the `nlopt-ocaml` and `nlopt` package can be seamless for users?

**Comment by UnixJunkie** [comment]
Ok, I'll have a look at that; probably today.

**Comment by UnixJunkie** [comment]
Ok, I added a transition package in there.
If someone can read its opam file, that would be nice.
Especially, do we really need a depends on OCaml line?

File: packages/nlopt-ocaml/nlopt-ocaml.transition/opam [hunk 1]
```diff
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+maintainer: "Francois Berenger"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/ocaml/dune"
+bug-reports: "https://github.com/ocaml/dune/issues"
+dev-repo: "git+https://github.com/ocaml/dune.git"
+license: "MIT"
+depends: [ "ocaml" ]
```
Comment (mseri) [inline packages/nlopt-ocaml/nlopt-ocaml.transition/opam:8]:
```suggestion
depends: [
  "ocaml"
  "nlopt"
]
```

**Comment by mseri** [comment]
Thanks!

**Comment by UnixJunkie** [comment]
Thanks for maintaining opam-repository, you are both doing a great job.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**